x/cdp/keeper/auctions.go

```diff
@@ -109,7 +109,11 @@ func (k Keeper) RunSurplusAndDebtAuctions(ctx sdk.Context) error {
 	remainingDebt := k.GetTotalDebt(ctx, types.LiquidatorMacc)
 	params := k.GetParams(ctx)
 	if remainingDebt.GTE(params.DebtAuctionThreshold) {
-		_, err := k.auctionKeeper.StartDebtAuction(ctx, types.LiquidatorMacc, sdk.NewCoin("usdx", remainingDebt), sdk.NewCoin(k.GetGovDenom(ctx), remainingDebt.Mul(sdk.NewInt(dump))), sdk.NewCoin(k.GetDebtDenom(ctx), remainingDebt))
+		debtLot := sdk.NewCoin(k.GetDebtDenom(ctx), params.DebtAuctionLot)
+		debtCoveredByLot := (sdk.NewDecFromInt(params.DebtAuctionLot).Quo(sdk.NewDecFromInt(remainingDebt))).Mul(sdk.NewDecFromInt(remainingDebt)).RoundInt()
```

Comment:
nit, can you add a comment here with the formula? Is it the right one?
`debtAuctionLot*remainingDebt/remainingDebt`

Reply:
Yah I totally second this 👍